agent/test-agent-cli/examples/example_test_agent_cli/Dockerfile

```diff
@@ -4,13 +4,9 @@ FROM ${BUILDER_IMAGE} as build
 
 ARG ARCH
 USER root
-# We need these environment variables set for building the `openssl-sys` crate
-ENV PKG_CONFIG_PATH=/${ARCH}-bottlerocket-linux-musl/sys-root/usr/lib/pkgconfig
-ENV PKG_CONFIG_ALLOW_CROSS=1
-ENV CARGO_HOME=/src/.cargo
```

Comment:
Still need this line for adding the cargo home?

Reply:
It was there twice.